charts/pomerium/README.md

```diff
@@ -298,6 +304,32 @@ A full listing of Pomerium's configuration variables can be found on the [config
 
 ## Upgrading
 
+### 10.0.0
+
+- All shared configuration has been moved from ENV vars to a configuration file.  Users of `config.existingSecret` must specify **all** options in their secret or leverage `extraEnv` to pass in
+  overrides.  
+  
+  Some of the impacted chart values are listed below:
+
+  - `authenticate.idp.provider`
+  - `authenticate.idp.url`
+  - `authenticate.cacheServiceUrl`
+  - `authenticate.idp.scopes`
+  - `config.insecure`
+  - `proxy.authenticateServiceUrl`
+  - `proxy.authorizeInternalUrl`
+  - `config.rootDomain` - Impacts ENV vars:
```

Comment:
Typo?  ` - Impacts ENV vars:` looks dangling?

Reply:
No...but maybe should rethink how to convey that.